assemblies/resources/bin/setenv

```diff
@@ -35,4 +35,9 @@
 # export KARAF_REDIRECT    # Enable/set the std/err redirection when using bin/start
 
 export EXTRA_JAVA_OPTS="-Dorg.eclipse.jetty.server.Request.maxFormContentSize=1500000 -Dfile.encoding=UTF-8"
+export JAVA_MAX_MEM=1G
+export JAVA_PERM_MEM=256M
 export KARAF_NOROOT=true
+
+
+. bin/check_ports "$KARAF_ETC"
```

Comment:
I think overwriting existing arguments is bash specific and may not work in different environments. See http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#dot

You actually do not need to pass this argument at all but since you also have access to `"$KARAF_ETC"` in `check_ports` if it is set here.

Reply:
While I'm all for POSIX compliant scripts, Opencast (or Karaf) requires `bash` for quite some time already (see shebang of `stop-opencast`). Not sure if this is intentional, but I think it used to be `sh`.

Reply:
fixed the parameter passing, can't do anything about needing bash as i'm using bash variable mangling